release/upload_casync_channel.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+CASYNC_DIR="${CASYNC_DIR:=/tmp/casync}"
+
+OPENPILOT_CHANNELS="https://commadist.blob.core.windows.net/openpilot-channels/"
```

Comment:
there's still too many things called "channels". conceptually, this is the "openpilot releases" bucket.